docs/plone-deployment/new-project.md

```diff
@@ -142,3 +144,10 @@ When the process completes successfully, it will exit with a message similar to
 ```console
 ✨  Done in 98.97s.
 ```
+
+````{note}
+Due to an output difference when translations are built by the `@plone/generator-volto`, at the moment, it is necessary to run on the root directory:
+
+```shell
+make i18n
```

Comment:
Close the triple backtick. Take care when nesting MyST blocks.
````suggestion
make i18n
```
````

Reply:
My intuition told me to close that backtick, but since the HTML version looked fine, I didn't. It's done now!